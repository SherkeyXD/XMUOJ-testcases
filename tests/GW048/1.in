2
12
24
abcd
ef