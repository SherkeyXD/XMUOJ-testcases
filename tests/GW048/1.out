12
24
abcd
ef